README.md

```diff
@@ -36,7 +34,6 @@ Supported variables are as follows:
 - restart_on_change - Changes to configuration or installed versions, will result in a restart if true.  Defaults to true.
 - daemon_args - Applicable to version 1.x of beats.  Allows run time params to be passed to beats.
 - logging_conf - Logging configuration.  Should be defined as a map - see Example playbook below.  Map is serialized into logging section of beat config. Defaults to "{"files":{"rotateeverybytes":10485760}}"
-- shipper_conf - Shipper configuration. Should be defined as a map - see Example playbook below. Map is serialized into shipper section of beat config. Defaults to "".
```

Comment:
Is there a specific reason to remove this option? Even though the default value is empty it would break backwards compatibility for anybody already using it.